src/components/input/bl-input.css

```diff
@@ -45,11 +45,30 @@
   --icon-size: var(--bl-font-body-text-3-line-height);
 }
 
-.wrapper:has(input:autofill) {
+.input-wrapper {
+  --border-size: 1px;
+
+  display: flex;
+  box-sizing: border-box;
+  gap: var(--padding-vertical);
+  height: var(--height);
+  border: solid var(--border-size) var(--border-color);
+  padding: 0 calc(var(--padding-horizontal) - var(--border-size));
+  border-radius: 4px;
+}
+
+:host([disabled]) .input-wrapper {
+  cursor: not-allowed;
+  background-color: var(--bl-color-secondary-background);
```

Comment:
I think it's now wise to have a `--input-background-color` variable. Like:

```css
.wrapper {
   --input-background-color: var(--bl-color-content-contrast);
  ....
}

.input-wrapper {
  background-color: var(--input-background-color):
  ...
}

.wrapper:has(input:autofill) {
  --background-color: var(--autofill-bg-color);
}
```

Then you can also use `--input-background-color` for half of the floating label in case we decide to use in that way.

Reply:
I didn't get what is meant here, input doesn't have a background as I mentioned at other comment.

Reply:
Let's consider fixing this in #462